Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/PrefabPublicHandler.cpp

```diff
@@ -724,13 +729,13 @@ namespace AzToolsFramework
             }
 
             PrefabDom beforeState;
+            m_instanceDomGeneratorInterface->GenerateEntityDom(beforeState, *entity);
```

Comment:
Won't this generate a dom for the entity after it has already changed?

Reply:
Never mind. Didn't realize the new method GenerateEntityDom. Still reviewing :).

Reply:
yeah I think the names need to be changed to something more specific, e.g. GenerateEntityDomFromFocus. I usually need to double-check the interface name to see which version it is calling. The other one is GenerateDomForEntity from serialization.